settings.yml

```diff
@@ -0,0 +1,20 @@
+---
+lib_name: lib_xassert
+project: '{{lib_name}}'
+title: '{{lib_name}}: XMOS Lightweight Assertions Library'
+version: 4.2.0
+
+documentation:
+  exclude_patterns_path: doc/exclude_patterns.inc
+  substitutions_path: doc/substitutions.inc
+  root_doc: doc/rst/index.rst
+  linkcheck_ignore_regex: ['']
```

Comment:
You can also add `cognidox_part_number` for docs and sw using these:
http://cognidox.xmos.local/cgi-perl/part-details?partnum=XM-006382-UG
http://cognidox.xmos.local/cgi-perl/part-details?partnum=XM-007507-SM